src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -133,27 +152,43 @@ public void sendBuildMessage(StepParameters stepParameters) {
     }
 
     private Card createJobStartedCard() {
-
-        factsBuilder.addStatusStarted();
-        factsBuilder.addStartTime();
-        factsBuilder.addRemarks();
-        addScmDetails();
-
         String jobName = run.getParent().getDisplayName();
-        String activityTitle = "Update from " + jobName + ".";
-        String activitySubtitle = "Latest status of build #" + run.getNumber();
-        Section section = new Section(activityTitle, activitySubtitle, factsBuilder.collect());
-
+        String activityTitle = "";
+        String activitySubtitle = "";
+        Section section;
+
+        if (isCompactNotification()) {
+            Parser parser = new Parser();
+            processScmDetails(false);
+            int fileCount = 0;
+            if (affectedFiles != null) 
+            {
+                fileCount = affectedFiles.size();
+            }
+            String uri = DisplayURLProvider.get().getRunURL(run);
+            activityTitle = jobName + " - <a href=\"" + uri + "\">#" + run.getNumber() + "</a> Started";
```

Comment:
No link to PR in compact mode? can be retrieved by pullRequestActionable or a modified version of it.

Reply:
Forgot about pull requests. There needs to be a link somewhere, you are right